gtsfm/evaluation/metrics.py

```diff
@@ -167,9 +169,8 @@ def _create_summary(self, data: np.ndarray) -> Dict[str, Any]:
         """
         if data.ndim != 1:
             raise ValueError("Metric must be a 1D distribution to get summary.")
-        if data.size == 0 or np.isnan(data).all():
+        if data.size == 0:  # or np.isnan(data).all():
```

Comment:
why is the NaN handling removed?

Reply:
I removed this while debugging and forgot to add it back in. I reverted it.